app/src/main/java/com/rahobbs/todo/activities/TodoListActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.rahobbs.todo.activities;
+
+import android.support.v4.app.Fragment;
+
+import com.rahobbs.todo.fragments.TodoListFragment;
+
+/**
+ * Created by rachael on 2/2/16.
+ * Activity that hosts the to-do list fragment
+ */
+public class TodoListActivity extends SingleFragmentActivity {
```

Comment:
So I know the Big Nerd Ranch has you write everything as Fragments but I think that sometimes that Fragments aren't the right approach and actually lead you down a path of total code bloat. Like here for example, you end up with this basically empty Activity for no reason. Also in this case, by using a Fragment, where you could just use the Activity you're adding an extra layer between the true "state" of the app and your code. And the less layers you are transmitting / sharing state over, the better.